### DISCUSSION TITLE
stale-while-revalidate with no seconds?

### DISCUSSION THREAD

**Discussion by olaj** [vercel/next.js#24272] [Help] [answered]
I'm having trouble forcing Cloudflare to cache the static json files generated by ISSG. I don't understand why, its the files generated on these pages https://alternativeto.net/category/productivity/

But one thing that i got confused about is that it's no value in seconds set to the stale-while-revalidate cache header. In all documentation i find it is supposed to be a value there i.e stale-while-revalidate=60 and so on.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control

Is it some special trick used in Next.js or is this actually a bug? It seems tested and so on if i look in the codebase so i suppose it should be like this. But in that case i reallt want some help to understand why these files aren't cached by Cloudflare.

**Comment by leerob** [comment, marked as answer]
https://github.com/vercel/next.js/pull/70674
